sentry-ruby/spec/sentry/configuration_spec.rb

```diff
@@ -238,6 +238,19 @@
           end
         end
       end
+
+      context "when having an error detecting the release" do
```

Comment:
Is this related to the new debug mode?

Reply:
@rhcarvalho no, it's just an uncovered scenario I found during the earlier commits 😅